compute/errors.go

```diff
@@ -29,6 +30,7 @@ import (
 )
 
 func ExtractProgramErrors(err error) (result []model.ProgramError) {
+	sentry.CaptureException(err)
```

Comment:
I'm not sure these are errors we want to catch in Sentry, they are related to the program parsing and execution.